conda_forge_tick/migrators.xsh

```diff
@@ -442,6 +442,8 @@ class Noarch(Migrator):
                         '  build:\n    - pip',
```

Comment:
This should be in host

Reply:
Build might not exist, but pip should still be added to host.

Reply:
ah ok I should reverse the setup that is there now.